.changeset/famous-ads-count.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-uikit/design-system': minor
+'@commercetools-uikit/link': minor
+---
+
+Adding inherit property for the links in the new theme
```

Comment:
**nit**: perhaps we don't need to be so specific and we can use something like:
```
Add new design token for `Link` component.
```

Reply:
Yes, that sounds better. Thank you. 

- [chore: adding changeset 2](https://github.com/commercetools/ui-kit/pull/2330/commits/2a2fcc35ddd6ace763aa9b854d0e27baec534688)

Reply:
We just need one changeset file so perhaps you can remove the first one (`famous-ads-count`).

Reply:
Also, I think `patch` would also suffice here instead of `minor` as we're simply refactoring some internal stuff (no new features or something similar).